### PR TITLE
Update Readme.md for FHIR path Rule support

### PR DESCRIPTION
**Pull request by skywalker-wu**
Update Readme.md for FHIR path Rule support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -236,31 +236,29 @@ Example usage to anonymize FHIR resource files in a folder:
 
 ## Configuration file format
 
-The configuration is specified in JSON format. It has three high-level sections. Two of these sections, namely _pathRules_, and _typeRules_ are meant to specify de-identification methods for data elements. De-identification configuration specified in the _pathRules_ section override the corresponding configurations in the _typeRules_ section. The third section named _parameters_ affect global behavior.
+The configuration is specified in JSON format. It has two high-level sections. One of these sections, namely _fhirPathRules_ is meant to specify de-identification methods for data elements. The second section named _parameters_ affect global behavior.
```
Comment (ranvijaykumar) [inline README.md:239]: Typo: It should be 'affects', not 'affect'.

File: README.md [hunk 2]
```diff
@@ -236,31 +236,29 @@ Example usage to anonymize FHIR resource files in a folder:
...
   "parameters": {
     "dateShiftKey": "",
     "enablePartialAgesForRedact": true
 }
 ```
 
-### Path Rules
-Path rules are key-value pairs that can be used to specify the de-identification methods for individual elements. Ex:
+### FHIR Path Rules
+FHIR path rules can be used to specify the de-identification methods for individual elements. Ex:
```
Comment (ranvijaykumar) [inline README.md:258]: 'for individual elements as well as elements of specific data types.' ?

File: README.md (hunk 1 above)
Comment (skywalker-wu) [reply README.md:239]: Thanks, updated.

File: README.md [hunk 3]
```diff
@@ -270,19 +268,9 @@ The elements can be specified using [FHIRPath](http://hl7.org/fhirpath/) syntax.
...
-"date": "dateShift"
-```
-The datatypes can be any of the [FHIR datatypes](https://www.hl7.org/fhir/datatypes.html). The method can be one from the following table.
-
-|Method| Applicable to | Description
-| ----- | ----- | ----- |
-|keep|All datatypes | Retains the value as is. |
-|redact|All datatypes| Removes the element. See the parameters section below to handle special cases. |
-|dateShift|date, dateTime, and instant datatypes | Shifts the value using the [Date-shift algorithm](#date-shift-algorithm).
+2 extension methods can be used in FHIR path rule to simplify the FHIR path:
```
Comment (ranvijaykumar) [inline README.md:271]: Replace 2 by Two.

File: README.md [hunk 4]
```diff
@@ -270,19 +268,9 @@ The elements can be specified using [FHIRPath](http://hl7.org/fhirpath/) syntax.
...
+- nodesByType('_typename_'): return descendants of type '_typename_', nodes in bundle resource and contained list would be excluded.
```
Comment (ranvijaykumar) [inline README.md:272]: Typo: return -> Returns.
Replace comma after '_typename_' with period.

File: README.md [hunk 5]
```diff
@@ -270,19 +268,9 @@ The elements can be specified using [FHIRPath](http://hl7.org/fhirpath/) syntax.
...
-The datatypes can be any of the [FHIR datatypes](https://www.hl7.org/fhir/datatypes.html). The method can be one from the following table.
-
-|Method| Applicable to | Description
-| ----- | ----- | ----- |
-|keep|All datatypes | Retains the value as is. |
-|redact|All datatypes| Removes the element. See the parameters section below to handle special cases. |
-|dateShift|date, dateTime, and instant datatypes | Shifts the value using the [Date-shift algorithm](#date-shift-algorithm).
+2 extension methods can be used in FHIR path rule to simplify the FHIR path:
+- nodesByType('_typename_'): return descendants of type '_typename_', nodes in bundle resource and contained list would be excluded. 
+- nodesByName('_name_'): return descendants of node name '_name_', nodes in bundle resource and contained list would be excluded.
```
Comment (ranvijaykumar) [inline README.md:273]: Typo: return -> Returns.
Replace comma after '_name_' with period.

File: README.md [hunk 6]
```diff
@@ -270,19 +268,9 @@ The elements can be specified using [FHIRPath](http://hl7.org/fhirpath/) syntax.
...
-| ----- | ----- | ----- |
-|keep|All datatypes | Retains the value as is. |
-|redact|All datatypes| Removes the element. See the parameters section below to handle special cases. |
-|dateShift|date, dateTime, and instant datatypes | Shifts the value using the [Date-shift algorithm](#date-shift-algorithm).
+2 extension methods can be used in FHIR path rule to simplify the FHIR path:
+- nodesByType('_typename_'): return descendants of type '_typename_', nodes in bundle resource and contained list would be excluded. 
+- nodesByName('_name_'): return descendants of node name '_name_', nodes in bundle resource and contained list would be excluded. 
 
 ### Parameters
 Parameters affect the de-identification methods specified in the type rules and path rules.
```
Comment (ranvijaykumar) [inline README.md:276]: There are no type riles and path rules any more

File: README.md (hunk 2 above)
Comment (skywalker-wu) [reply README.md]: Updated. Thanks

File: README.md (hunk 6 above)
Comment (skywalker-wu) [reply README.md]: Removed path rule and type rule.

File: README.md (hunk 5 above)
Comment (skywalker-wu) [reply README.md]: Updated. thanks.

File: README.md (hunk 4 above)
Comment (skywalker-wu) [reply README.md]: Updated. Thanks

File: README.md (hunk 3 above)
Comment (skywalker-wu) [reply README.md]: Updated. thanks.

**Pull request closed without merging**